.werft/jobs/build/self-hosted-upgrade-tests.ts

```diff
@@ -81,9 +81,9 @@ export async function triggerUpgradeTests(werft: Werft, config: JobConfig, usern
 export async function triggerSelfHostedPreview(werft: Werft, config: JobConfig, username: string) {
     const replicatedChannel =  config.replicatedChannel || config.repository.branch;
     const cluster =  config.cluster || "k3s";
-    const formattedBranch = config.repository.branch.replace("/", "-").slice(0,10)
+    const formattedBranch = config.repository.branch.replace("/", "-").slice(0,11)
```

Comment:
What's the relevance of the `10`/`11`? Is that the length of a word that is being split by or something?

Reply:
I had a `x` added to the line below because I was trying to test redeploy and I wanted to get a different subdomain. It caused for an [unfortunate subdomain yesterday ](https://werft.gitpod-dev.com/job/gitpod-k3s-installer-tests-head.33)so I am removing that and keeping the subdomain name length the same.